js/main.js

```diff
@@ -103,7 +103,7 @@ requirejs.config({
 		"negative-controls": "components/negative-controls",
 		"d3": "d3.min",
 		"d3_tip": "d3.tip",
-		"jnj_chart": "jnj.chart",
+		"jnj_chart": "https://cdn.rawgit.com/odysseusinc/atlascharts/master/index",
```

Comment:
As the charts functionality is required by both Atlas and Achilles, we moved it out into a separate module and therefore separate repo, and removed a local copy from Atlas'es repo (to make these projects reference the lib, not copy). In current state the library is just copy-pasted, which leads to unsync of code base in the projects using it, makes you to do double work when changes are required and is just a bad programming practice.